test/main.js

```diff
@@ -61,6 +61,25 @@ describe('glob-stream', function() {
       });
     });
 
+    it('should handle ( ) in directory paths', function(done) {
+      var cwd = join(__dirname, './fixtures/has (parens)');
```

Comment:
I can add more tests or update some of the existing ones that already test that to include the `test.dmc`.